[CI] AttachmentProcessorTests.testIndexedCharsWithResourceName() failure

**Build scan**: https://gradle-enterprise.elastic.co/s/terl76fif5hmm/tests/:plugins:ingest-attachment:test/org.elasticsearch.ingest.attachment.AttachmentProcessorTests/testIndexedCharsWithResourceName?expanded-stacktrace=WyIwIl0#1

**Repro line**: `./gradlew ':plugins:ingest-attachment:test' --tests "org.elasticsearch.ingest.attachment.AttachmentProcessorTests.testIndexedCharsWithResourceName" -Dtests.seed=BDC087E6CE6DF6C -Dtests.security.manager=true -Dtests.locale=es-DO -Dtests.timezone=Europe/Moscow -Druntime.java=11`

**Reproduces locally?**: No (but was on macos)

**Applicable branches**: master
(windows ci only)

**Failure history**: Started to fail since 8 december

https://gradle-enterprise.elastic.co/scans/tests?search.relativeStartTime=P7D&search.timeZoneId=Europe/Amsterdam&tests.container=org.elasticsearch.ingest.attachment.AttachmentProcessorTests&tests.sortField=FAILED&tests.test=testIndexedCharsWithResourceName&tests.unstableOnly=true

**Failure excerpt**:

```
java.lang.AssertionError: |  
-- | --
  | Expected: a string containing "碩鼠よ碩鼠よ、\n我が黍を食らう無かれ！" |  
  | but: was "碩鼠よ碩鼠よ、 |  
  | 我が黍を食らう無かれ！ |  
  | 三歳女に貫えしに、 |  
  | 我を肯じて顧みる莫し。 |  
  |   |  
  | 逝きて将に女を去り、 |  
  | 彼の楽土へ適かん。 |  
  | 楽土よ楽土よ、 |  
  | 爰に我が所を得ん。 |  
  |   |  
  | 碩鼠よ碩鼠よ、" |  
```

--- a/plugins/ingest-attachment/src/test/java/org/elasticsearch/ingest/attachment/AttachmentProcessorTests.java
+++ b/plugins/ingest-attachment/src/test/java/org/elasticsearch/ingest/attachment/AttachmentProcessorTests.java
@@ -358,7 +358,6 @@ public class AttachmentProcessorTests extends ESTestCase {
 
     }
 
-    @AwaitsFix(bugUrl = "https://github.com/elastic/elasticsearch/issues/66086")
     public void testIndexedCharsWithResourceName() throws Exception {
         processor = new AttachmentProcessor(randomAlphaOfLength(10), null, "source_field",
             "target_field", EnumSet.allOf(AttachmentProcessor.Property.class), 100,
@@ -384,7 +383,7 @@ public class AttachmentProcessorTests extends ESTestCase {
         attachmentData = parseDocument("text-cjk-euc-jp.txt", processor, Collections.singletonMap("max_length", 100), true);
 
         assertThat(attachmentData.keySet(), containsInAnyOrder("language", "content", "content_type", "content_length"));
-        assertThat(attachmentData.get("content").toString(), containsString("碩鼠よ碩鼠よ、\n我が黍を食らう無かれ！"));
+        assertThat(attachmentData.get("content").toString(), containsString("碩鼠よ碩鼠よ、" + System.lineSeparator() + "我が黍を食らう無かれ！"));
         assertThat(attachmentData.get("content_type").toString(), containsString("text/plain"));
         assertThat(attachmentData.get("content_type").toString(), containsString("charset=EUC-JP"));
         assertThat(attachmentData.get("content_length"), is(100L));

